config/boards/armsom-sige5.csc

```diff
@@ -2,14 +2,15 @@
 BOARD_NAME="ArmSoM Sige5"
 BOARDFAMILY="rk35xx"
 BOOTCONFIG="armsom-sige5-rk3576_defconfig"
-KERNEL_TARGET="rk3576"
+KERNEL_TARGET="vendor"
```

Comment:
This change is not allwinner related?

Reply:
This commit came in by mistake but as its still draft ... working on it.